conda_forge_tick/utils.py

```diff
@@ -20,6 +20,7 @@
 import jinja2.sandbox
 import networkx as nx
 import ruamel.yaml
+from conda_recipe_v2_schema.model import SimpleRecipe
```

Comment:
This package will need to be added to conda-forge and added to the environment.yaml file here. Not sure what the status is for the first thing.

Reply:
Yes, that's what I did :)
Do I have to update the lockfile as well?

Reply:
yes there is a script that will relock things.